composer.json

```diff
@@ -1,6 +1,5 @@
 {
     "name": "phile-cms/phile",
-    "version": "1.6.0",
```

Comment:
Please take a look at the documentation:
https://getcomposer.org/doc/04-schema.md#version

> Optional if the package repository can infer the version from somewhere, such as the VCS tag name in the VCS repository. In that case it is also recommended to omit it.

So the trick is, to tag the version in github